sdk/mongocluster/arm-mongocluster/test/public/mongoclusterOperations.spec.ts

```diff
@@ -144,7 +141,7 @@ describe("MongoCluster test", () => {
 
   it("mongoClusters delete test", async function () {
```

Comment:
https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/mongocluster/arm-mongocluster/test/public/mongoclusterOperations.spec.ts#L131

tags are empty object, can we remove them?

Reply:
I think update the resource with a tag is a very simple case, and It's definition is Record<string, string>, so I just use a empty object to make it exist